gbmodel/model_sqlalchemy.py

```diff
@@ -182,6 +184,26 @@ def get_team_members(self, tid):
             return None
         return mems
 
+    def get_students_in_session(self, session_id):
```

Comment:
this method needs comments and error handling

Reply:
Done